### ISSUE TITLE
Help wanted reagarding api usage

### ISSUE THREAD

**Issue by Razamindset** [sigma67/ytmusicapi#706]
Hi there ! I was recently experimenting with this great api. I just wanted to ask a quick question. I am working on a web app and I want to use this api but i cannot figure out how to get songs url that i can actually use for playback. Is there any way to get urls/links. Do i need to use authenticated requests or there is now way? Any help will be appreciated.

**Comment by Utkarsh-vishnoi** [comment]
I don't think you will directly get the streaming/playback URL using this API.

One alternative could be [this](https://github.com/yt-dlp/yt-dlp#:~:text=%23%20Stream%20the%20video%20being%20downloaded%20to%20stdout%0A%24%20yt%2Ddlp%20%2Do%20%2D%20BaW_jenozKc).

**Comment by sigma67** [comment]
You can use ``get_song`` to get relevant information. Check out @KoljaWindeler profile
